core/integration/steps/amm_submission.go

```diff
@@ -101,9 +102,10 @@ func parseSubmitAMMTable(table *godog.Table) []RowWrapper {
 
 func parseAmendAMMTable(table *godog.Table) []RowWrapper {
 	return StrictParseTable(table, []string{
-		"party",     // str
-		"market id", // str
-		"slippage",  // dec
+		"party",        // str
+		"market id",    // str
+		"slippage",     // dec
+		"proposed fee", // str
```

Comment:
is this going to be required for an amend? And shouldn't this be a decimal, rather than a string?

Reply:
For the sake of test is better for it to be required.